Design-Docs/tools/image-md-check.py

```diff
@@ -8,8 +8,11 @@
 # (c) Kurt Garloff <kurt@garloff.de>, 09/2022
 # SPDX-License-Identifier: CC-BY-SA-4.0
 
-import os, sys, getopt, time
+import getopt
 import openstack
+import os
+import sys
+import time
```

Comment:
this should've been a separate commit. no need to change now, but for future work.